srtcore/queue.h

```diff
@@ -533,6 +533,7 @@ struct CMultiplexer
    CChannel* m_pChannel;    // The UDP channel for sending and receiving
    CTimer* m_pTimer;        // The timer
 
+   uint32_t m_piIP[4];
    int m_iPort;         // The UDP port number of this multiplexer
```

Comment:
Instead of this, please remove `m_iPort` and use the `sockaddr_any` type from `netinet_any.h` header. The `m_iVersion` distinction method will soon be removed.